src/api/plugins/video/warren_video/api.py

```diff
@@ -37,7 +41,32 @@ async def views(
     else:
         indicator = DailyVideoViews(**indicator_kwargs)
     try:
-        response = Response[VideoViews](
+        response = Response[VideoStats](
+            status=StatusEnum.SUCCESS, content=indicator.compute()
+        )
+    except KeyError as exception:
+        logger.error(exception)
+        return Response[Error](
+            status=StatusEnum.FAILED, content=Error(error_message=str(exception))
+        )
+    return response
+
+
+@router.get("/{video_uuid:path}/downloads")
+async def downloads(
+    video_uuid: IRI,
```

Comment:
I think we need to rename this argument everywhere (in a separated PR) as in the xAPI semantic, the video identifier is an IRI that can be a UUID in the form of an IRI `uuid://xxx...`, but it's not an UUID.

```suggestion
    video_id: IRI,
```

Reply:
https://github.com/openfun/warren/pull/48